### PR TITLE
Thorny bush hazard

### PR DESCRIPTION
**Pull request by AMacLeod99**

### CODE REVIEW

File: Content/Blueprints/Enemies/Hazards/BP_StingingBush.uasset
Comment (Saretzky) [inline Content/Blueprints/Enemies/Hazards/BP_StingingBush.uasset:1]: I would move the hazards folder from enemies to the level objects folder

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by Saretzky — APPROVED** [review]
nice

**Review by CrazyTobi — APPROVED** [review]
Looks good

**Review by MysticAmazon — APPROVED** [review]
everything seems good now !

**Pull request merged**